halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -70,4 +74,19 @@ public void accept(ProblemSetBuilder psBuilder, Validator v) {
         .map(NamedContext::getName)
         .collect(Collectors.toList());
   }
+
+  protected List<String> dockerRegistriesOptions(ProblemSetBuilder psBuilder) {
```

Comment:
I realize this is named funny (`dockerRegistryOptions` sounds better), but the reflection for option creation requires the prefix of the variable name to match the options method.